### PR TITLE
HR1 Procedures and Functionality changes

### PR DESCRIPTION
**Pull request by ajtheb**

### CODE REVIEW

**Review by Naman2018157 — APPROVED** [review]
Reveiwed

File: FusionIIIT/templates/dashboard/sidenavbar.html [hunk 1]
```diff
@@ -172,25 +172,3 @@ <h4> File Tracking </h4>
...
-            </a>
-            {% comment %}The Activity Log Item ends here!{% endcomment %}
-
-            {% comment %}The Logout Item starts here!{% endcomment %}
-            <a class="centered raised item" href='/accounts/logout'>
-                <span class="inverted header">Logout</span>
-                <i class="large inverted sign out icon right floated"></i>
-            </a>
-            <div class="item"></div>
-            {% comment %}The Logout Item ends here!{% endcomment %}
```
Comment (garg3133) [inline FusionIIIT/templates/dashboard/sidenavbar.html:191]: Why??????????????

**Review by garg3133 — COMMENTED** [review]
Why have you deleted the Meet our Team and Logout button from sidenavbar?????

**Comment by garg3133** [comment]
image

Don't you even test the code before raising the PR? You've even deleted the `{% endblock %}` 🤦‍♂️

**Pull request merged**